uk_bin_collection/uk_bin_collection/councils/CrawleyBoroughCouncil.py

```
@@ -86,27 +84,68 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                    })
+
+        # Try XML format if rows_data didn't yield results
+        if not bindata["bins"] and xml_data:
+            try:
+                root = ET.fromstring(xml_data)
+                fields = root.findall(".//Field")
+                field_names = [f.get("Name") for f in fields]
+                rows = root.findall(".//Row")
+                for row in rows:
+                    values = [c.text or "" for c in row]
+                    row_dict = dict(zip(field_names, values))
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)